.coveragerc

```diff
@@ -1,12 +0,0 @@
-[run]
-# don't collect the plugins subdirectories
-omit =
-    plugins/*/env/*.py
-    plugins/conftest.py
-    plugins/*/*/tests/*
-    tests/plugins/*
-    sideboard/tests/*
-    tests/*
-
-include =
-    sideboard/*
```

Comment:
Perhaps a controversial decision; I moved `.coveragerc` into `setup.cfg` just to cut down on the number of files in the project directory. I'm happy to put it back if anyone objects!